be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -80,10 +82,9 @@ std::shared_ptr<RowsetReader> AlphaRowset::create_reader() {
             LOG(WARNING) << "alpha rowset load failed. rowset path:" << _rowset_path;
             return nullptr;
         }
-        set_loaded(true);
```

Comment:
why do you drop this line?

Reply:
because init() above has called set_loaded(true);